src/MICore/LaunchOptions.cs

```diff
@@ -888,6 +921,18 @@ protected set
             }
         }
 
+        private ReadOnlyCollection<PathMapEntry> _sourceMap;
```

Comment:
If we are going to call it that, I am assuming the more important place to fix it is in the XSD. Correct?

---
In reply to: [98104661](https://github.com/Microsoft/MIEngine/pull/541#discussion_r98104661) [](ancestors = 98104661)